missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Game.java

```diff
@@ -332,12 +332,22 @@ public void playerJoinInGame(Player player, boolean isSpectatorJoin) {
 
             Team team = getNextTeam();
             team.addMember(mwPlayer);
-            player.sendMessage(Messages.getMessage("team_assigned").replace("%team%", team.getFullname()));
-
-            broadcast(Messages.getMessage("lobby_joined")
-                    .replace("%max_players%", Integer.toString(getLobby().getMaxSize()))
-                    .replace("%players%", Integer.toString(getPlayers().values().size()))
-                    .replace("%player%", player.getName()));
+            player.sendMessage(Messages.getMessage("team.team_assigned").replace("%team%", team.getFullname()));
+            
+            if (state == GameState.LOBBY) {
+                broadcast(Messages.getMessage("lobby.player_joined")
+                        .replace("%max_players%", Integer.toString(getLobby().getMaxSize()))
+                        .replace("%players%", Integer.toString(getPlayers().values().size()))
+                        .replace("%player%", player.getName())
+                        .replace("%team%", team.getFullname()));
+            } else if (state == GameState.INGAME) {
+                broadcast(Messages.getMessage("game.player_joined")
+                        .replace("%max_players%", Integer.toString(getLobby().getMaxSize()))
+                        .replace("%players%", Integer.toString(getPlayers().values().size()))
+                        .replace("%player%", player.getName())
+                        .replace("%team%", team.getFullname()));
+            }
```

Comment:
We have a lot of code duplication here. I would recommend something like the following:
```java
String message;
if (state == GameState.LOBBY) {
  message = Messages.getMessage("lobby.player_joined");
} else if (state == GameState.INGAME) {
  message = Messages.getMessage("game.player_joined");
}

broadcast(message.replace()...);
```